samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp

```diff
@@ -48,9 +49,9 @@
    </div>
    <div class="divRight col-md-8 col-lg-6 col-sm-7 col-xs-12">
       <h:inputText id="IE_hidden" value="" disabled="true" style="display: none;" />
-      <h:inputText id="filteredUser" value="#{eventLog.filteredUser}" size="30" autocomplete="off"
-         onfocus="resetHelpValue('#{eventLogMessages.search_hint}', 'eventLogId:filteredUser')"
-         onclick="resetHelpValue('#{eventLogMessages.search_hint}', 'eventLogId:filteredUser')"/>
```

Comment:
What's the purpose of this `resetHelpValue()` method. Are we losing anything by removing it?

Reply:
Hey, the purpose of `resetHelpValue()` is to clear the hint from the input when it's focused. 
Originally there wasn't any hint showing, and I had trouble trying to implement it.
So based in other tools, I added the "Search" label. And removed the mentioned method.

Reply:
Thanks for explaining!